### ISSUE TITLE
if aria-value/now/min/max is a decimal value, warning "ARIA state and property values must be valid" 

### ISSUE THREAD

**Issue by akhoury** [GoogleChrome/accessibility-developer-tools#182]
According to the spec, a decimal value is fine. What am I missing?

http://www.w3.org/TR/wai-aria/states_and_properties#aria-valuenow

> The value of aria-valuenow is a decimal number. If the range is a set of numeric values, then aria-valuenow is one of those values. For example, if the range is [0, 1], a valid aria-valuenow is 0.5. A value outside the range, such as -2.5 or 1.1, is invalid.

### `aria-valuenow` and `aria-valuemax` warnings
![screen shot 2015-07-11 at 12 40 00 pm](https://cloud.githubusercontent.com/assets/1398375/8634523/1325739e-27ca-11e5-84a9-12a3c7d1aa0a.png)

### `aria-valuemin` warning
![screen shot 2015-07-11 at 12 33 41 pm](https://cloud.githubusercontent.com/assets/1398375/8634517/8db89114-27c9-11e5-97e6-dd6a3d78fb41.png)

Thank you!

**Comment by akhoury** [comment]
is there a reason why both `integer` and 'decimal` values are validated the same way?

https://github.com/GoogleChrome/accessibility-developer-tools/blob/48b283ed84ab8544095b31c4b2052d523f9ffd8a/src/js/AccessibilityUtils.js#L692-L707

```javascript
    case "integer":
    case "decimal":
        var validNumber = axs.utils.isValidNumber(value);
        if (!validNumber.valid) {
            result.valid = false;
            result.reason = validNumber.reason;
            return result;
        }
        if (Math.floor(validNumber.value) != validNumber.value) { // <--- this is testing decimals as well
            result.valid = false;
            result.reason = '' + value + ' is not a whole integer';
        } else {
            result.valid = true;
            result.value = validNumber.value;
        }
        return result;
```

Looks like an easy fix

**Comment by RickSBrown** [comment]
Looks like the intention was to fall through from `number` to `decimal` not `integer` to `decimal`.

**Comment by RickSBrown** [comment]
PR #196 will fix this.

**Comment by RickSBrown** [comment]
Closed by #196
